### PR TITLE
feat(analytics): Add nullable field description to analytics table

### PR DESCRIPTION
**Pull request by gh-PonyM**
Implements #1953

### CODE REVIEW

**Review by winged — CHANGES_REQUESTED** [review]
Nice work, let's see what CI thinks :)

The commit doesn't look like [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) yet, but otherwise LGTM!

**Review by winged — APPROVED** [review]

**Comment by DadaDaMotha** [comment]
Running pytest in parallel causes the issue. As you pointed out last time, running them with `poetry run pytest -n 0 --reuse-db` inside docker passes the tests. What should we do in this case?

File: caluma/caluma_analytics/filters.py [hunk 1]
```diff
@@ -53,7 +50,7 @@ class AnalyticsTableOrderSet(FilterSet):
     meta = MetaFieldOrdering()
     attribute = AttributeOrderingFactory(
         models.AnalyticsTable,
-        fields=["created_at", "modified_at", "slug", "name"],
+        fields=["created_at", "modified_at", "slug", "name", "description"],
```
Comment (winged) [inline caluma/caluma_analytics/filters.py:53]: Does it make sense to allow ordering by description? No need to fix, I just don't think it's particularly useful

File: caluma/caluma_analytics/tests/test_table.py [hunk 2]
```diff
@@ -71,15 +83,25 @@ def test_create_table(db, snapshot, schema_executor):
...
     )
 
     assert not result.errors
-    assert models.AnalyticsTable.objects.filter(pk="test-table").exists()
+    item = models.AnalyticsTable.objects.filter(pk="test-table").first()
+    assert str(item.description) == "Foo bar"
 
     snapshot.assert_match(result.data)
+    assert result.data["saveAnalyticsTable"]["analyticsTable"]
+    print(result.data)
```
Comment (winged) [inline caluma/caluma_analytics/tests/test_table.py:98]: Print should be removed in "production" code

**Review by winged — CHANGES_REQUESTED** [review]
Some minor cleanup required, otherwise LGTM

File: caluma/caluma_analytics/filters.py (hunk 1 above)
Comment (open-dynaMIX) [reply caluma/caluma_analytics/filters.py:53]: Sorry, should have marked as `draft`. Wanted to first fix CI, that's why I pushed early, even before I took a closer look at the changes :sweat_smile:

**Review by winged — APPROVED** [review]

**Pull request merged**